bench-vortex/src/reader.rs

```diff
@@ -134,47 +126,77 @@ pub fn write_csv_as_parquet(csv_path: PathBuf, output_path: &Path) -> VortexResu
     Ok(())
 }
 
-pub async fn take_vortex(path: &Path, indices: &[u64]) -> VortexResult<Array> {
-    let mut file = tokio::fs::File::open(path).await?;
-
-    file.seek(SeekFrom::End(-8)).await?;
-    let footer_len = file.read_u64_le().await? as usize;
-
-    file.seek(SeekFrom::End(-(footer_len as i64 + 8))).await?;
-    let mut footer_bytes = BytesMut::with_capacity(footer_len);
-    unsafe { footer_bytes.set_len(footer_len) }
-    file.read_exact(footer_bytes.as_mut()).await?;
+pub async fn read_vortex_footer_format<R: VortexReadAt>(
```

Comment:
This should be translated to vortex footer format and lifted out of benchmarks. Will happen in next pr

Reply:
The format will be more complicated than just this